andela/travel_tool_back
# 161096201 Send travel survey email notification upon checkout

#### What does this PR do?
Send an email notification to a user once they check out of a guest house
#### Description of Task to be completed?

- Update the mail template for a case where a user needs to fill travel survey
- Add a method that enables notification engine to send out an email with a survey when a trip status changes to `check out`
-Write test to ensure a method is called to send the survey email

#### How should this be manually tested?
Clone the repo - `git clone https://github.com/andela/travel_tool_back.git`
Enter into the project directory - `cd travel_tool_back`
checkout the branch `ft-send-travel-survey-161096201`
Install dependencies using `yarn install`
Rollback and run database migrations with `yarn db:rollmigrate`
Start the server with `make start`
Navigate to travela in your browser and login
Create a new request and set yourself as the approver for the request.
Approve the created request as a manager
On your Database GUI, in the `Trips` table select a bed for your created trip
On Travela UI, navigate to the `Residence` tab and click on `checkin`
On the same page click on `check out` to check out of a guest house
You should receive an email notification requesting you to fill a survey email after check-out

#### Any background context you want to provide?
To be able to test this feature, ensure that you add `SURVEY_URL=https://www.surveygizmo.com/s3/4345421/Andela-Global-Staff-Travel-Survey` to your `.env` file
#### What are the relevant pivotal tracker stories?
[161096201](https://www.pivotaltracker.com/story/show/161096201)
#### Screenshots (if appropriate)
![image](https://user-images.githubusercontent.com/37342753/47154707-b4f67b80-d2eb-11e8-8971-dafcfd0e28e0.png)


#### Questions:
N/A
